src/cdk-experimental/menu/menu-item.ts

```diff
@@ -125,7 +125,7 @@ export class CdkMenuItem implements FocusableOption, FocusableElement, OnDestroy
     }
 
     // don't set the tabindex if there are no open sibling or parent menus
-    if (!event || (event && !this._getMenuStack().isEmpty())) {
+    if (!event || (event && !this._getMenuStack()?.isEmpty())) {
```

Comment:
I think the `event &&` here is redundant since it's assumed by the `!event` being true before.